api/interfaces/init.go

```diff
@@ -1,15 +1,41 @@
 package interfaces
 
-import "go.mongodb.org/mongo-driver/bson/primitive"
+import (
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
 
+type Coordinates struct {
```

Comment:
This file no longer exists, please move this struct to the `api/interfaces/requests.go` file.